frontend/public/components/pod.tsx

```diff
@@ -664,26 +664,22 @@ export const PodStatus: React.FC<PodStatusProps> = ({ pod }) => {
       headerTitle = t('public~Pod crash loop back-off');
       const containers: ContainerSpec[] = pod.spec.containers;
       footerLinks = (
-        <div>
+        <div className="pf-c-content">
```

Comment:
Would probably be better to use `<TextContent>` and `<Text component={TextVariants.p}>` here so PatternFly is controlling this.

Reply:
Agreed and updated.